### PR TITLE
Fix: Vagrantfile rubocop violations

### PR DESCRIPTION
**Pull request by thefliik**
This pull introduces/changes:
 * fixes rubocop violations in Vagrantfile

Pings:
@cheerfulstoic
@subvertallchris

### CODE REVIEW

**Comment by thefliik** [comment]
these changes are also included in #1419

**Comment by cheerfulstoic** [comment]
:+1: Thanks

File: Vagrantfile [hunk 1]
```diff
@@ -5,31 +5,31 @@
...
 
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine and only allow access
   # via 127.0.0.1 to disable public access
-  config.vm.network "forwarded_port", guest: 7474, host: 7474, host_ip: "127.0.0.1"
-  config.vm.network "forwarded_port", guest: 7473, host: 7473, host_ip: "127.0.0.1"
-  config.vm.network "forwarded_port", guest: 7472, host: 7472, host_ip: "127.0.0.1"
+  config.vm.network 'forwarded_port', guest: 7474, host: 9474, host_ip: '127.0.0.1'
+  config.vm.network 'forwarded_port', guest: 7473, host: 9473, host_ip: '127.0.0.1'
+  config.vm.network 'forwarded_port', guest: 7472, host: 9472, host_ip: '127.0.0.1'
```
Comment (thefliik) [inline Vagrantfile]: I should not have committed this host port change

Comment (cheerfulstoic) [reply Vagrantfile]: Cool, I just put them back

**Pull request merged**